test.js

```diff
@@ -1,8 +1,31 @@
 import test from 'ava';
 import execa from 'execa';
 
+function randomName() {
+	return `asdasfgrgafadsgaf${Math.random().toString().slice(2)}`;
+}
+
 test(async t => {
-	const pkgName = `asdasfgrgafadsgaf${Math.random().toString().slice(2)}`;
-	const ret = await execa('./cli.js', [pkgName, '--color'], {cwd: __dirname});
+	const ret = await execa('./cli.js', [randomName(), '--color'], {cwd: __dirname});
 	t.regex(ret.stdout, /is available/);
 });
+
+test(async t => {
+	const pkgName = 'chalk';
+	try {
+		const ret = await execa('./cli.js', [pkgName, '--color'], {cwd: __dirname});
+		t.regex(ret.stdout, /is unavailable/);
+	} catch (err) {
+		t.ok(err);
```

Comment:
I believe this test is incorrect. If `execa` throws here (which I believe it doesn't), the `t.regex` will never be executed and vice versa.

```js
test('unavailable', async t => {
	const ret = await execa('./cli.js', ['chalk', '--color'], {cwd: __dirname});
	t.regex(ret.stdout, /chalk is unavailable/);
});
```